src/DetailPage.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react'
+import fetch from 'superagent'
+import PokeList from './PokeList.js'
+
+export default class DetailPage extends Component {
+state = {
+    pokemon: []
+}
+
+componentDidMount = async () => {
+    const response = await fetch.get(`https://alchemy-pokedex.herokuapp.com/api/pokedex/?pokemon=${this.props.match.params.pokemon}`)
+
+    this.setState({ pokemon: response.body.results })
```

Comment:
You have an edgecase where if you click on charizard, you will get charizard, charizard-mega-x and charizard-mega-y. You can 
``` this.setState({ pokemon: response.body.results[0] })
```
but then you would need to change your <PokeList> to display properly since it be an object not an array. Just a thought and not really an issue.